rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -996,6 +996,54 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+        logger.warn("[processValidationFailure] Proposed federation validation failed so svp values will be cleared.");
+        clearSvpValues();
+    }
+
+    private void clearSvpValues() {
+        federationSupport.clearProposedFederation();
+
+        String methodName = "[clearSvpValues]";
+
+        Optional<Sha256Hash> svpFundTxHashUnsigned = provider.getSvpFundTxHashUnsigned();
+        if (svpFundTxHashUnsigned.isPresent()) {
+            logger.warn("{} Fund tx {} was never registered.", methodName, svpFundTxHashUnsigned.get());
```

Comment:
I am bit confused with these logs. If the fund tx was never registered why is the value present?

Reply:
But i will modify it to say "Fund tx **change**" to avoid confusion. Thanks!